pfdp/css/cards.css

```diff
@@ -124,4 +157,12 @@
   .pill img {
     margin-right: 12px;
   }
+  
+  .testimonials {
+    display: grid;
```

Comment:
no need to repeat the values which stay the same. i think only `grid-template-columns changes` here

Reply:
removed the necessary values